cluster/src/main/java/com/hartwig/pipeline/tertiary/orange/Orange.java

```diff
@@ -51,9 +51,10 @@
 import com.hartwig.pipeline.tertiary.virus.VirusInterpreterOutput;
 import com.hartwig.pipeline.tools.Versions;
 
-@Namespace(Orange.NAMESPACE)
+@Namespace(Orange.NAMESPACE_NO_GERMLINE)
```

Comment:
This is an intellij refactor gone wrong but @nedleitch how this annotation actually used? Is it possible to have 2 namespaces for one stage or do i need to subclass?

Reply:
That's certainly not the intended usage but clearly the annotation does not enforce this. The idea was that these annotations would replace the constants that most of them are referencing, ultimately strengthening the contract on the stages and paving the way for a declarative stage runner. We didn't really push too hard so now its value is limited.

It _is_ involved with automated testing to ensure that all stages are mentioned in at least one starting point. That was added for the Protect reruns to ensure that when a new stage is added it won't get run by that starting point, which we'd hit if you were to subclass. Again maybe the value is limited.